students/jesse_miller/session09/donor_models.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+'''
+Let's try this by following the directions.  So, this is going to be where the
+donor manipulation occurs.  The goal is to compartmentalize everything so that
+functions can be called by other programs if and when needed.
+'''
+
+
+class Donor:
+    '''
+    This will be slightly easier in a sense because we're only dealing with a
+    single donor.  However, I get to look forward to pairing everything down to
+    an interesting, if not insane, degree from previous attempts.
+    '''
+
+    def __init__(self, name):
+        self.name = name
+        self.donations = [] #?  We'll try it.
```

Comment:
This is ok. In a more complex scenario, this one could become a list of _Donation_ objects, with more details about each donation.